src/pages/instructions/instructions-page.ts

```diff
@@ -0,0 +1,122 @@
+import { LitElement, css, html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { ICON_CHEVRON_SLOTTED, PAGE_STYLES, WAKE_WORDS } from "../../const";
+import "@material/web/list/list";
+import "@material/web/list/list-item";
+import "@material/web/icon/icon";
+import "../../components/card-layout";
+import "../../components/error-notice";
+import { createRecorder } from "../../util/recorder";
+import type { Recorder } from "../../util/recorder";
+
+@customElement("instructions-page")
+export class InstructionsPage extends LitElement {
+  @property() public startRecording!: (recorder: Recorder) => void;
+
+  @property() public description!: string;
+
+  @property() public wakeWord!: string;
+
+  @property({
+    attribute: false,
+  })
+  public recorderError: boolean = false;
```

Comment:
You want this to be `@state` I think? It's not set externally. 
```suggestion
  @state
  private recorderError: boolean = false;
```